Lib/gftools/axisregistry/grade.textproto

```diff
@@ -1,33 +1,17 @@
 # GRAD based on https://github.com/TypeNetwork/Opentype-1.8-Axis-Proposal/blob/gh-pages/proposal/Grade/ProposalSummary.md
 tag: "GRAD"
 display_name: "Grade"
-min_value: -1
-max_value: 1
+min_value: -1000
+max_value: 1000
 default_value: 0
-precision: -1
-fallback {
-  name: "Full Negative"
-  value: -1
-}
-fallback {
-  name: "Half Negative"
-  value: -0.5
-}
+precision: 0
 fallback {
   name: "Normal"
   value: 0
 }
-fallback {
-  name: "Half Positive"
-  value: 0.5
-}
-fallback {
-  name: "Full Positive"
-  value: 1
-}
 description:
   "Finesse the style from lighter to bolder in typographic color, by varying"
   " stroke thicknesses or other forms, without any changes overall width,"
   " inter-letter spacing, or kerning, so there are no changes to line breaks"
   " or page layout. Negative grade makes the style lighter, while positive"
-  " grade makes it bolder. "
+  " grade makes it bolder. The units are the same as in the Weight axis."
```

Comment:
"The units are the same as in the Weight axis"

Is this true when wght has an avar?

@dberlow @davelab6